crate/guides/fetch.md

```diff
@@ -59,15 +59,15 @@ fn view(model: &Model) -> Node<Msg> {
     )]
 }
 
-fn init(_: Url, orders: &mut impl Orders<Msg>) -> Init<Model> {
+fn after_mount(_: Url, orders: &mut impl Orders<Msg>) -> AfterMount<Model> {
     orders.perform_cmd(fetch_data());
-    Init::new(Model::default())
+    AfterMount::default()
 }
 
 #[wasm_bindgen]
 pub fn render() {
-    let app = seed::App::build(init, update, view)
-        .finish()
+    let app = seed::App::builder(update, view)
+        .after_mount(after_mount)
         .run();
 
     app.update(Msg::FetchData);
```

Comment:
Todo for next PRs

In all examples:
- `seed::App` replace with  `App` and make sure we have the line `use seed::{*, prelude::*}` at the top of the example (if necessary) ; And we probably should add `App` into `prelude`.
- `let app ...  app.update(... ` is anti-pattern. It should be refactored with `after_mount`.

Reply:
Updated this PR and the Rust quickstart with this import style.